From 6ea596e39cb9eb988b7ef2f056e610f55b68967c Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Thu, 14 Apr 2016 15:23:52 +0300
Subject: [PATCH 119/239] fix: tools: a3700: Remove clocks setup from DDR-800
 descriptor

- Remove redundant clocks setup from DDR-800 TIM descriptor.

Change-Id: I78a2d30d2c34eb50d624071a41a16b1a216897e0
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/29059
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Tested-by: Wilson Ding <dingwei@marvell.com>
---
 tools/wtp/ddr-800.txt | 50 --------------------------------------------------
 1 file changed, 50 deletions(-)

diff --git a/tools/wtp/ddr-800.txt b/tools/wtp/ddr-800.txt
index 89cde60..c3225d2 100644
--- a/tools/wtp/ddr-800.txt
+++ b/tools/wtp/ddr-800.txt
@@ -1,53 +1,3 @@
-; Switch all clocks to REFCLOCK
-WRITE: 0XC0013010 0x00000000
-WRITE: 0XC0018010 0x00000000
-; TBG-A: SE vco_div 0x0,
-; DIFF vco_div 0x1, vco_range 0xa
-; tbg_N 0x30 KVCO = 1600 MHz
-WRITE: 0xC0013204 0x00C00091
-WRITE: 0xC0013204 0x00C000C1
-WRITE: 0xC0013220 0x08030803
-WRITE: 0xC0013208 0x94011400
-WRITE: 0xC0013230 0x00020002
-WRITE: 0xC0013208 0x94011400
-WRITE: 0xC001320C 0x53E556E6
-WRITE: 0xC0013210 0x014A014A
-WRITE: 0xC001320C 0x53E556E5
-WRITE: 0xC0013204 0x00C000C0
-WRITE: 0xC0013208 0x94011400
-WAIT_FOR_BIT_SET: 0xC0013208 0x80008000 0x00100000
-DELAY: 0x00000100
-; TBG-B: SE vco_div 0x1,
-; DIFF vco_div 0x2, vco_range 0xb
-; tbg_N 0x3c KVCO = 2000 MHz
-WRITE: 0xC0013204 0x00C100C0
-WRITE: 0xC0013204 0x00F100C0
-WRITE: 0xC0013220 0x08030803
-WRITE: 0xC0013208 0x14019400
-WRITE: 0xC0013230 0x00040002
-WRITE: 0xC0013208 0x14019400
-WRITE: 0xC001320C 0x56E556E5
-WRITE: 0xC0013210 0x014B014A
-WRITE: 0xC001320C 0x56E656E5
-WRITE: 0xC0013204 0x00F000C0
-WRITE: 0xC0013208 0x14019400
-WAIT_FOR_BIT_SET: 0xC0013208 0x80008000 0x00100000
-DELAY: 0x00000100
-; Set clocks to 800/800 preset
-WRITE: 0xC0013014 0x00000000
-WRITE: 0xC0013004 0x1296202C
-WRITE: 0xC0013008 0x21061AA9
-WRITE: 0xC001300C 0x20543082
-WRITE: 0xC0013000 0x0003C0F2
-WRITE: 0xC0018014 0x00180000
-WRITE: 0xC0018004 0x02515508
-WRITE: 0xC0018008 0x00307880
-WRITE: 0xC001800C 0x00000540
-WRITE: 0xC0018000 0x003F8F40
-WRITE: 0xC0013210 0x0014B014A
-; Switch all clocks to back dividers
-WRITE: 0xC0013010 0x00009FFF
-WRITE: 0xC0018010 0x000007AA
 ; Set DDR to 800MHz
 ; Pre-init
 WRITE: 0xC0014008 0x00404500
-- 
1.9.1

